code/game/objects/structures/crates_lockers/closets/bsdroppod.dm

```diff
@@ -0,0 +1,84 @@
+//The "BDPtarget" temp visual is created by the expressconsole, which in turn makes two things: a falling droppod animation, and the droppod itself.
+
+
+//------------------------------------BLUESPACE DROP POD-------------------------------------//
+/obj/structure/closet/bsdroppod
+	name = "Bluespace Drop Pod"
+	desc = "A Nanotrasen supply drop pod."
+	icon = 'icons/obj/2x2.dmi'
+	icon_state = "BDP"
+	pixel_x = -16//2x2 sprite
+	pixel_y = -5
+	pixel_z = 0
+	layer = TABLE_LAYER//so that the crate inside doesn't appear underneath
+	allow_objects = TRUE
+	allow_dense = TRUE
+	delivery_icon = null
+	can_weld_shut = FALSE
+	armor = list("melee" = 30, "bullet" = 50, "laser" = 50, "energy" = 100, "bomb" = 90, "bio" = 0, "rad" = 0, "fire" = 80, "acid" = 80)
+	anchored = TRUE
+	anchorable = FALSE
+	var/datum/supply_order/SupplyOrder
+
+/obj/structure/closet/bsdroppod/Initialize(mapload, datum/supply_order/so)
+	. = ..()
+	SupplyOrder = so//uses Supply Order passed from expressconsole into BDPtarget
+	addtimer(CALLBACK(src, .proc/open), 30)//open 3seconds after appearing
+
+/obj/structure/closet/bsdroppod/update_icon()//called in initialize, all we have to do is add a door icon
+	cut_overlays()//just incase update_icon somehow gets called multiple times, prevents overlay spam
+	add_overlay("BDP_door")
+
+/obj/structure/closet/bsdroppod/attackby(obj/item/W, mob/user, params)//i no need to be welded or anchored or whatever
+	return
```

Comment:
You shouldn't override attackby like this, it needs to call the parent proc always. Find another way to do what you want